subset/security/brute_test/src/main/java/SetupTest.java

```diff
@@ -0,0 +1,60 @@
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SetupTest {
+
+  String host;
+  int connectionPort;
+  String macAddress;
+  SshSocket sshSocket = null;
+  Map<String, String> macDevices = new HashMap<String, String>();
+  boolean debug = false;
+  static final int minimumMACAddressLength = 5;
+
+  public void readLocalFile() {
```

Comment:
Again, thinking about the 'next dev' (probably still you) -> this function reads a MacList, it's a very special local file - could the function naming be made more explicit?